content/webapp/components/Exhibition/Exhibition.tsx

```diff
@@ -156,7 +169,7 @@ function getAccessibilityItems(exhibition: UiExhibition): ExhibitionItem[] {
       description: [
         {
           type: 'paragraph',
-          text: exhibition.accessContentOverride || defaultAccessContent,
+          text: 'Large-print guides, transcripts and magnifiers are available in the gallery',
```

Comment:
@alexwlchan I heard you before – agree these bits of microcopy should probably live elsewhere.

Reply:
👉 https://github.com/wellcomecollection/wellcomecollection.org/issues/7602